setup.py

```diff
@@ -58,6 +59,16 @@
     ],
 }
 
+console_scripts = list(itertools.chain.from_iterable([
+    [
+        'sphinx-build%s = sphinx.cmd.build:main' % version,
+        'sphinx-quickstart%s = sphinx.cmd.quickstart:main' % version,
```

Comment:
Is this also needed? quickstart is not related with python version.

Reply:
I just converted everything to be consistent. I can drop this if you prefer?